tests/parser/types/test_bytes.py

```diff
@@ -4,6 +4,19 @@
 )
 
 
+def test_convert_from_bool(get_contract_with_gas_estimation):
+    exp_code = """
+@public
+def testConvertBytes32(flag: bool) -> bytes32:
+    flagBytes: bytes32 = convert(flag, bytes32)
+    return flagBytes
+    """
+
+    c = get_contract_with_gas_estimation(exp_code)
+    assert c.testConvertBytes32(False) == b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00"
+    assert c.testConvertBytes32(True) == b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01"
```

Comment:
I think just slicing `c.testConvertBytes32(True)[-1] == b'x01'` here would be more readable. If we want make sure about the 32 byte padding perhaps just an assert len() == 32 as well.